conf/test.config

```diff
@@ -20,6 +20,5 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
```

Comment:
yup! I'm making a new test data set right now.

Reply:
i like mixing the test data set for long or short to quickly test if they all run, but I think what I might do is have different full profiles so that the output reports will be more demonstrative of what a user could expect to see for their real data. for that it would be good to have pacbio, nanopora, and illumina. i'm not super concerned with getting this situated right now, but would be good in the long run potentially.